firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java

```diff
@@ -70,7 +70,7 @@
   public void updateSession(PerfSession session) {
     // Note(b/152218504): Being defensive to fix the NPE
     if (session == null) {
-      logger.info("Unable to add new SessionId to the Network Trace. Continuing without it.");
+      logger.debug("Unable to add new SessionId to the Network Trace. Continuing without it.");
```

Comment:
What criteria have you used to determine that this should be debug instead of warn?

Reply:
My criteria:
 - Debug: Useful message for debugging purposes. For example, the value of a remote config variable.
 - Warning: Unexpected behavior. The process might be continued, but take extra caution.

Therefore, I think this one should be `Warning`. Updated the PR :)

Reply:
Thanks! Left a few comments based on that criteria :)